### ISSUE TITLE
Возможность отключать raw data

### ISSUE THREAD

**Issue by optozorax** [klavarog/keyboard-layout-editor#4]
Когда передвигаешь клавиши, по всей видимости, в вкладку raw data полностью перезаписывается новый текст, и поэтому kle так страшно лагает. Нужно убрать это записывание, чтобы либо его можно было вырубить, либо чтобы оно делалось по специальной кнопке. Потому что с текущим методом сериализации оно страшно лагает.

**Comment by bouncepaw** [comment]
а, так вот откуда лаги :) Да, надо будет вырубить вон

**Comment by bouncepaw** [comment]
Да, действительно на каждое движение raw data заново делается. Поскольку мы решили выкинуть raw data вон, было легко и перестать каждый раз заново делать raw data: [строчка](https://github.com/klavarog/keyboard-layout-editor/commit/2ef84a0ef4327d2f552ba17bc08a00430ed8125b#diff-1a6ac1d403028e61ffcc496f05f050dfR673)

По ощущениям действительно стало быстрее. Проверишь? Хочу закрыть ишью.

**Comment by optozorax** [comment]
Ишью закрывается только тогда, когда вмержится в мастер. Я ещё я бы хотел это проверить на тестовом сервере

**Comment by bouncepaw** [comment]
хм, мерж не стоит делать, потому что вместе с этим будет много breaking changes. Значит ишью будет висеть открытой, пока не закончу frog-edition-ветку :)

**Comment by optozorax** [comment]
Нет, надо каждое изменение для каждого issue делать в отдельной ветке.

**Comment by optozorax** [comment]
Оказывается нифига не пофиксилось

**Comment by bouncepaw** [comment]
Ну, всё же поровнее двигаются клавиши, меньше кадров пропускается, но нельзя сказать, что они плавно ходят, тут надо глубже копать. Вероятно, можно закрыть именно этот ишью и открыть новый? :thinking: Проблема-то шире, а raw data отключена.

**Comment by optozorax** [comment]
согласен, закрою ишью когда создатшь новое
